recipes-support/bt-6lowpan-setup/bt-6lowpan-setup_0.1.bb

```diff
@@ -0,0 +1,32 @@
+SUMMARY = "Setup BT 6LoWPAN network / modules"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit allarch
+
+SRC_URI = " \
+	file://bt-6lowpan.network.in \
+	file://modules-6lowpan.conf \
+"
+
+S = "${WORKDIR}"
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+# Allow build time customizations by the user
+BT_6LOWPAN_INTERFACE ?= "bt0"
+BT_6LOWPAN_NETWORK ?= ""
+
+do_compile() {
+	if [ ! -z "${BT_6LOWPAN_NETWORK}" ]; then
+		sed -e 's/@@BT_6LOWPAN_NETWORK@@/${BT_6LOWPAN_NETWORK}/' \
+			${WORKDIR}/bt-6lowpan.network.in > ${WORKDIR}/bt-6lowpan.network
+	fi
+}
+
+do_install() {
+	if [ -f "${WORKDIR}/bt-6lowpan.network" ]; then
+		install -m 0644 ${WORKDIR}/bt-6lowpan.network ${IMAGE_ROOTFS}${exec_prefix}/lib/systemd/network/60-bt-6lowpan.network
```

Comment:
When installing just install to ${D}${systemd_unitdir}/network... instead, as this is part of the recipe (it was before in IMAGE_ROOTFS because it was part of the recipe responsible for building the image).